apps/defines/rvc_defines.h

```diff
@@ -0,0 +1,4 @@
+#define D_MEM_BASE   0x00004000
+#define CR_MEM_BASE  0x00007000
+#define VGA_MEM_BASE 0x00008000
+#define FP_RESULTS   0x00004f00
```

Comment:
That's it?
What about all the different CRs?
Memory regions?

Reply:
Taken care of.